sources/packages/web/src/views/aest/institution/ApproveDenyDesignation.vue

```diff
@@ -1,33 +1,75 @@
 <template>
-  <!-- CSS class temporary-modal is work around for lack of responsiveness of v-dialog. -->
-  <modal-dialog-base
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-    :title="title"
-  >
-    <template v-slot:content>
-      <div class="temporary-modal">
-        <formio
-          formName="approvedenydesignation"
-          :data="designation"
-          @loaded="formLoaded"
-          @submitted="submitDesignationUpdate"
-        ></formio>
-      </div>
-    </template>
-    <template v-slot:footer>
-      <check-permission-role :role="Role.InstitutionApproveDeclineDesignation">
-        <template #="{ notAllowed }">
+  <v-form ref="approveDenyDesignation">
+    <modal-dialog-base :showDialog="showDialog" :title="title" max-width="730">
+      <template #content>
+        <error-summary :errors="approveDenyDesignation.errors" />
+        <span class="label-value">{{ subTitle }} </span>
+        <content-group class="my-3" v-if="showApprovalContent">
+          <span class="label-bold color-blue">Effective dates</span>
+          <v-row
+            ><v-col
+              ><v-text-field
+                label="Start date"
+                class="mt-2"
+                type="date"
+                placeholder="yyyy-MM-dd"
+                v-model="formModel.startDate"
+                variant="outlined"
+                :rules="[checkStringDateFormatRule]"
+            /></v-col>
+            <v-col
+              ><v-text-field
+                label="Expiry date"
+                class="mt-2"
+                type="date"
+                placeholder="yyyy-MM-dd"
+                v-model="formModel.endDate"
+                variant="outlined"
+                :rules="[checkStringDateFormatRule]" /></v-col></v-row
+          ><v-divider></v-divider
+          ><span class="label-bold color-blue">Designate locations</span
+          ><template
+            v-for="(item, index) in formModel.locationsDesignations"
+            :key="index"
+          >
+            <v-list-item :value="index" class="mt-2 ml-n4">
+              <v-list-item-title>
+                <v-row
+                  ><v-col
+                    ><p>
+                      <span class="label-bold">{{ item.locationName }}</span>
+                    </p>
+                    <span>{{ item.locationAddress }}</span></v-col
+                  ><v-col
+                    ><v-checkbox
+                      class="float-right"
+                      v-model="item.approved"
+                      hide-details="auto" /></v-col
+                ></v-row>
+              </v-list-item-title>
+            </v-list-item> </template></content-group
+        ><v-textarea
+          class="mt-4"
+          label="Notes"
+          placeholder="Long text..."
+          v-model="formModel.note"
+          variant="outlined"
+          :rules="[checkNotesLengthRule]"
+        />
+      </template>
+      <template #footer>
+        <check-permission-role
```

Comment:
if its purpose is to disable the btn , then you need to extend the template ` <template #="{ notAllowed }">`
![image](https://user-images.githubusercontent.com/77353155/197876082-085f180f-304c-458d-a546-923ff1e9a943.png)